Makefile

```diff
@@ -132,9 +135,10 @@ fixtures/%:
 	if [ ! -d "opentelemetry-helm-charts" ]; then \
 		git clone https://github.com/open-telemetry/opentelemetry-helm-charts.git; \
 	fi
-	helm install test -f .github/workflows/e2e/k8s/collector-helm-values.yml opentelemetry-helm-charts/charts/opentelemetry-collector
+	helm install test -f .github/workflows/e2e/k8s/$(FIXTURE_MODE)/collector-helm-values.yml opentelemetry-helm-charts/charts/opentelemetry-collector
+	sleep 5
```

Comment:
Fixture tests were failing locally without sleep. The next waits for the container to be ready, but sometimes the container isn't created and fails.

Reply:
We already have `kubectl` wait for condition below. Wait for condition is failing when i run this locally.

Reply:
The issue with adding sleep is that is going to be brittle (it may work on you machine, but fail on others) and they accumulate.

I'm surprised the `wait` command would fail, but I'm also curious if the `rollout` command works better for you.

Reply:
Converted to `until`